docs/source/install/vagrant.rst

```diff
@@ -0,0 +1,121 @@
+.. |br| raw:: html
+
+   <br />
```

Comment:
@LindsayHill Do you know any other place I can put this block, so we could re-use in all other `.rst` files?

Reply:
There's a `common/` directory in the install docs that I use for other blocks that get included in multiple places. Maybe there?

Reply:
I was thinking if there is a "maigc file" which is auto-included in all docs, without doing explicitly `include`.

Reply:
Found it in http://www.sphinx-doc.org/en/stable/config.html#confval-rst_prolog